recipes-utils/imx-misc-tools/files/imx-otp-tool-add-revocation-command.patch

```diff
@@ -0,0 +1,218 @@
+From: Alex Gonzalez <alexg@balena.io>
+Date: Mon, 9 Dec 2024 16:14:11 +0100
+Subject: [PATCH] imx-otp-tool: add revocation command
```

Comment:
Do you intend to upstream this?

Reply:
I was intending to after all testing is done - not sure if it has a lot of value as it's a one man project but it's the righ thing to try.